qutebrowser/browser/downloads.py

```diff
@@ -166,6 +166,7 @@ def get_filename_question(*, suggested_filename, url, parent=None):
     q.title = "Save file to:"
     q.text = "Please enter a location for <b>{}</b>".format(
         html.escape(url.toDisplayString()))
+    q.yank_text = url.toString()
```

Comment:
This should be `toDisplayString()` to not contain e.g. passwords.

Reply:
Also, that's a good start, but in other prompts there are things to yank as well. You might want to search for `Question(` and `message.ask` and `message.confirm` and think about what could be yanked there. Sometimes, nothing makes sense, often it's an URL ("Open external application" prompt, certificate errors, etc.), maybe there are places where it makes sense to yank other text?